docs/server/push-duty.rst

```diff
@@ -185,7 +187,7 @@ Once complete
 
 Create a new github document for the *next push*, for example:
 
-https://github.com/mozilla/addons/blob/master/releases/2018/08/16.md
+https://github.com/mozilla/addons/blob/master/releases/2019/05/09.md
```

Comment:
This wasn't really necessary, but 🤷‍♂

Reply:
Well, I was thinking that someone might just copy and paste from the one linked as an "example"